conf/messages.en

```diff
@@ -96,12 +97,12 @@ penalisedEstateAgentsActDetail.error.required = Enter a description of the penal
 penalisedEstateAgentsActDetail.error.length = Description of the penalty must be 255 characters or less
 penalisedEstateAgentsActDetail.error.punctuation = Description of the penalty can include letters a to z, numbers, spaces and punctuation
 
-penalisedProfessionalBody.title = Has the business, or anyone in the business, ever been penalised by a professional body for estate agency business activities?
-penalisedProfessionalBody.heading = Has the business, or anyone in the business, ever been penalised by a professional body for estate agency business activities?
+penalisedProfessionalBody.title = Penalties by a professional body
```

Comment:
The ATs need a change for this too

Reply:
PR for ATs when you have a sec - https://github.com/hmrc/amls-acceptance-test/pull/374